### ISSUE TITLE
Feature Request: Enable mutli-tab selection functionality

### ISSUE THREAD

**Issue by brian6932** [NiklasGollenstede/unload-tabs#39]
At the moment, when CTRL + Clicking multiple tabs and using an unload tabs function will only work on the tab that is focused (the one you're currently in), as opposed to your selected tabs. I think it would be pretty cool to have it work with selected tabs as well. For example, you can select which tabs you want as active, and then right click unload other, or select a bunch of active tabs and suspend them all together, less binary than 1 tab or all the other tabs, while still being able to keep the same context menus.

**Comment by FichteFoll** [comment]
I was under the impression that this was made possible via https://github.com/NiklasGollenstede/unload-tabs/pull/34, but it appears that this feature is currently not available.

I'd very much like to suspend multiple tabs at a time when I know I'm not going to use them in the near future.

**Comment by FichteFoll** [comment]
Never mind, I just realized that the change in #34 was never included in a release, only in https://github.com/NiklasGollenstede/unload-tabs/releases/tag/v0.8.6b30. It works there.

**Comment by brian6932** [comment]
Ah, in that case I'll just sideload the web-extension, thanks for the lmk.

**Comment by jpmhouston** [comment]
I'm not sure if Unload Tab chosen from the menu is affecting all selected tabs or not, but in any case it's made not clear. I don't even know if this functionality is in the version I have, if there's been a release in the last 18ish months, but in any case in the git change you mentioned I didn't notice anything specific to menu items.

I'd suggest some variation to the menu items when multiple tabs are selected (with the obvious functionality):

- in the tools sub menu, "Unload Tab" should become "Unload Selected Tabs"
- when one of the selected tabs is right clicked, if can you add 2 menu items then I'd suggest both "Unload Tab" and "Unload Selected Tabs"
- when the right clicked tabs isn't within the selected, I think the menu should show only "Unload Tab"

**Comment by brian6932** [comment]
@jpmhouston This is now built into Firefox, and it still uses `Unload tab` when multiple are selected.

**Comment by FichteFoll** [comment]
Not quite, it shows "Unload 2 tabs" for me when I have 2 tabs selected.

**Comment by brian6932** [comment]
Oh true, I didn't notice cause I always press <kbd>U</kbd>+<kbd>ENTER</kbd>.
